app/assets/javascripts/pageflow/editor/views/edit_meta_data_view.js

```diff
@@ -34,9 +34,16 @@ pageflow.EditMetaDataView = Backbone.Marionette.Layout.extend({
       });
 
       this.input('credits', pageflow.TextAreaInputView);
-      this.input('author', pageflow.TextInputView);
-      this.input('publisher', pageflow.TextInputView);
-      this.input('keywords', pageflow.TextInputView);
+
+      this.input('author', pageflow.TextInputView, {
+        placeholder: pageflow.theming.attributes.default_author
+      });
+      this.input('publisher', pageflow.TextInputView, {
+        placeholder: pageflow.theming.attributes.default_publisher
+      });
+      this.input('keywords', pageflow.TextInputView, {
+        placeholder: pageflow.theming.attributes.default_keywords
+      });
```

Comment:
I believe that's what my [latest change accomplished](https://github.com/scrollytelling/pageflow/blob/8bec49b35e67fdb47a0326834e49fa46cee4d798/app/views/pageflow/editor/themings/_theming.json.jbuilder#L8).